index.bs

```diff
@@ -1527,6 +1545,40 @@ example of the latter, when the user is accessing the [=[RP]=] from a given clie
 use a [=roaming authenticator=] which was originally registered with the [=[RP]=] using a different client.
 
 
+### Attestation Presentation Preference enumeration (enum <dfn enum>AttestationPresentationPreference</dfn>) ### {#attestation}
+
+[=[RPS]=] may use the {{AttestationPresentationPreference}} enum to specify their preference regarding attestation types used
+during credential generation.
+
+<pre class="idl">
+    enum AttestationPresentationPreference {
+        "none",
+        "indirect",
+        "direct"
+    };
+</pre>
+
+<ul>
+    <li><dfn>none</dfn> - indicates that the [=[RP]=] is not interested in attestation, and prefers fast and reliable credential
+      creation over obtaining a verifiable attestation. The client MAY replace the attestation statement generated
+      by the authenticator with a meaningless client-generated attestation statement in this case, for example in order to prevent
```

Comment:
we need to correspondingly update the [Registering a new credential](https://w3c.github.io/webauthn/#registering-a-new-credential) relying party operation such that in the case of a blinded attestation that the RP not try to validate it, yes?